deploy/operator/setup_lso.sh

```diff
@@ -18,7 +18,7 @@ function print_help() {
 function install_lso() {
   oc adm new-project openshift-local-storage || true
 
-  oc annotate project openshift-local-storage openshift.io/node-selector='' --overwrite=true
+  retry -- annotate project openshift-local-storage openshift.io/node-selector='' --overwrite=true
```

Comment:
shouldn't it be:
```suggestion
  retry -- oc annotate project openshift-local-storage openshift.io/node-selector='' --overwrite=true
```
?

Reply:
:man_facepalming:

Reply:
Fixed, I shouldn't touch anything today :sweat_smile: